### DISCUSSION TITLE
P:dialog ignores h:outputtext escape=false

### DISCUSSION THREAD

**Discussion by xterms** [primefaces/community#1631] [PrimeFaces] [answered]
Hello, 

So I have the following view: 
```
<p:dialog id="SimpleDialog" header="#{dialogBean.header}" widgetVar="SimpleDialog" class="modal" modal="#{dialogBean.modal}" visible="false" closable="#{dialogBean.closeAble}" >
        <div class="container"> 
			<h:outputText value="#{dialogBean.text}" escape="false" />
        </div>  
    </p:dialog>
```

And a backingbean that looks like this: 
```
public void showAppVersion() {
 String test = "<h1>testing</h1>";
  dialogBean.openSimpleDialog("Versionsinformation", test); 
}

public void openSimpleDialog(String header, String text) {
       PrimeFaces current = PrimeFaces.current();              //get primefaces instance
        current.ajax().update("SimpleDialog");                  //uppdate the viw with new data from backend
        current.executeScript("PF('SimpleDialog').show();");    //show the dialog window
}

public String getText() {
        return text;
    }
```

But the result of this is just
\<h1>testing\</h1>

If I put the <h:outputText value="#{dialogBean.text}" escape="false" /> anywhere outside the <p:dialog> it works as intended. Anyone that can see what I'm missing?

**Comment by melloware** [comment]
you didn't mention your version of PrimeFaces? 

Can you provide an executable example using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project?

**Comment by Santhanam12345** [comment]
Hi,
       First make sure that you have a valid variable in your bean class because here you have mentioned value="#{dialogBean.text}" but in your bean class no variable available in that name.Getter method only available for text variable. Second thing inside showAppVersion() method you declared String "test" variable. How bean class get the values from local variable.

in openSimpleDialog method the second parameter not used anywhere. So declare text variable as global and assign value to that text variable and check it.

**Comment by melloware** [comment, marked as answer]
Closing unless more info is provided
